tests/exit_methods.pyi

```diff
@@ -34,22 +35,24 @@ class FiveAndAHalf:
     def __exit__(self, typ: object, exc: builtins.object, tb: object) -> None: ...
     async def __aexit__(self, typ: object, exc: object, tb: builtins.object) -> None: ...
 
-def __exit__(foo: str, bar: int) -> list[str]: ...
-async def __aexit__(spam: bytes, eggs: dict[str, Any]) -> set[int]: ...
+class Six:
+    def __exit__(self, *args: Unused) -> bool: ...
+    def __aexit__(self, *args: Unused) -> Awaitable[None]: ...
```

Comment:
Could you also add a test for the fullname, i.e. `_typeshed.Unused`?

Reply:
I added that. And tests for "Unused" at different parameter location, since that's different code paths in the checks.